hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneClusterImpl.java

```diff
@@ -286,6 +312,54 @@ public OzoneClient newClient() throws IOException {
     return client;
   }
 
+  @Override
+  public AmazonS3 newS3Client() {
+    // TODO: Parameterize tests between Virtual host style and Path style
+    return createS3Client(true);
+  }
+
+  public AmazonS3 createS3Client(boolean enablePathStyle) {
```

Comment:
In a follow-up task I think we should add a separate factory class for creating the client, and remove `AmazonS3` from `MiniOzoneCluster` interface.

Reply:
Good idea, raised https://issues.apache.org/jira/browse/HDDS-11576